src/emulator/CMakeLists.txt

```diff
@@ -51,6 +51,8 @@ relocation.cpp
 relocation.h
 vpk.cpp
 vpk.h
+sfo.h
```

Comment:
So far I've been trying to keep file lists sorted, for a couple of reasons:
- Any ordering resists merge conflicts (and bad merges) better than no ordering
- Easier to compare to a directory listing